src/ppx_import.ml

```diff
@@ -465,79 +465,72 @@ let rec psig_of_tsig ~subst (tsig : Compat.signature_item_407 list) :
   | [] -> []
   | _ -> assert false
 
-let module_type ~tool_name ~input_name (modtype_decl : Ppxlib.module_type) =
+let module_type ~tool_name ~input_name
+    (package_type : Ppxlib__Import.package_type) =
```

Comment:
Hey @tatchi, have you seen this suggestion? It's just better to use the public API instead of the internal one, since there's no guarantee for the internal one to stay stable.